docs_dev/releasing.md

```diff
@@ -150,12 +151,9 @@ gcloud config set project kubeflow-releasing
 gcloud container clusters get-credentials kubeflow-releasing --zone us-central1-a --project kubeflow-releasing
 ```
 
-Use [this script](https://github.com/jlewi/kubeflow-dev/blob/master/create_context.sh) to set up your context properly:
-
-```
-create_context.sh $(kubectl config current-context) kubeflow-releasing
-```
+## Authorization to Publish a Release
 
+Need to join [release team](https://github.com/kubeflow/internal-acls/blob/1234654eb219e2c06ed5fdc2c4e662a02fccc740/github-orgs/kubeflow/org.yaml#L388) before you can publish a release.
 
```

Comment:
How about adding a section 

Getting ready to cut a release 

and adding the following

1. Create a P0 bug of kind process to track the major release.
1. Announce the release and link to the issue in the #release channel in kubeflow.slack.com

Reply:
done